TESTS/model/integration_test.cpp

```diff
@@ -84,22 +91,22 @@ TEST(Integration, run_10x_mem_check) {
 
 void compute_model(Tensor& input_10, Tensor& Identity0) {
   // start rendering local declare snippets
-  TFLM::QuantizeOperator<int8_t, float> op_000;
+  QuantizeOperator<int8_t, float> op_000;
 
-  QuantizedFullyConnectedOperator<int8_t> op_001(
-      TFLM::TfLiteFusedActivation::kTfLiteActRelu);
+  FullyConnectedOperator<int8_t> op_001(
+      TfLiteFusedActivation::kTfLiteActRelu);
```

Comment:
As you've proposed, the activation type can be abstracted for TFLM, uTensor reference, and maybe, ONNX and beyond.